kafka-rest/src/main/java/io/confluent/kafkarest/RestCustomRequestLog.java

```diff
@@ -0,0 +1,51 @@
+package io.confluent.kafkarest;
+
+import org.eclipse.jetty.server.CustomRequestLog;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.RequestLog;
+import org.eclipse.jetty.server.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * RestCustomRequestLog implements Jetty's RequestLog interface. It offers the same log-format
+ * jetty's CustomRequestLog. Additionally, it would append configured request-attributes (see
+ * requestAttributesToLog) to the end of the log.
+ */
+public class RestCustomRequestLog implements RequestLog {
```

Comment:
I think we should make the name a little detailed, like `CustomRequestLogWithAttributes`

Reply:
I will rename to `CustomLog` to be concise, the logging of `request-attibutes` is implementation detail that comments will explain.